Tella/Data/Networking/Models/UwaziLanguageResult.swift

```diff
@@ -13,19 +13,16 @@ import Foundation
 struct UwaziLanguageResult: Codable {
     let rows: [UwaziLanguageRow]?
 }
-
 // MARK: - Row
 struct UwaziLanguageRow: Codable, Hashable {
```

Comment:
Could you apply the mapping to all the data models ?

Reply:
Done